modules/ROOT/partials/configuration/valid_elements.adoc

```diff
@@ -7,14 +7,14 @@ This option contains a comma separated list of element conversion chunks. Each c
 
 If you just want to add or change some behavior for a few items, use the xref:extended_valid_elements[`extended_valid_elements`] option
```

Comment:
This line has a ref that refers to something in another file, it's better to include the full ref here:
```suggestion
If you just want to add or change some behavior for a few items, use the xref:configure/content-filtering.adoc#extended_valid_elements[`extended_valid_elements`] option
```